templates/terraform/custom_flatten/group_id_to_name.erb

```diff
@@ -0,0 +1,4 @@
+func flattenMonitoringUptimeCheckConfigResourceGroupGroupId(v interface{}, d *schema.ResourceData) interface{} {
```

Comment:
This is fine for now, but we should start thinking about doing something about this pattern- there's a few similar flatteners around.